site/en/docs/crux/release-notes/index.md

```diff
@@ -43,6 +43,41 @@ The CrUX dataset on BigQuery is generally updated on the second Tuesday of every
 
 In the list below, we've curated some release notes for each monthly dataset. Subscribe to our [CrUX Announce](https://groups.google.com/a/chromium.org/forum/#!forum/chrome-ux-report-announce) mailing list or follow [@ChromeUXReport](https://twitter.com/ChromeUXReport) on Twitter for release Announcements.
 
+## 202209
+
+[Announcement](TBC)
+
+Publication date
+ : October 11, 2022
+
+What's new
+ : - The CrUX API now includes `collectionPeriod` object showing what day's data is included in the response as shown below. More details in the [CrUX API docs](./api/#collection-period).
+
+```json
+    "collectionPeriod": {
+      "firstDate": {
+        "year": 2022,
+        "month": 10,
+        "day": 9
+      },
+      "lastDate": {
+        "year": 2022,
+        "month": 9,
+        "day": 12
+      }
+    }
+```
+
+Notable stats
+ : - 17,715,277 origins
+ : - 41.4% of origins have good [Core Web Vitals](https://web.dev/vitals/#core-web-vitals)
+
+Other recent CrUX-related updates:
+ : - A paper was published [Evaluating the Accuracy of Popular Website Lists](https://zakird.com/papers/toplists.pdf), which noted that [CrUX data is significantly more accurate than other data sources](https://twitter.com/zakirbpd/status/1573344656456310784?s=20&t=EmRR5_RVpK8N8VA2izgh_g).
+ : - The [web-vitals](https://github.com/GoogleChrome/web-vitals/) library released v3 which includes the new INP metric, and in-built attribution to help identify elements causing your metric numbers.
+ : - [Barry Pollard](https://twitter.com/tunetheweb) wrote [Why is CrUX data different from my RUM data?](https://web.dev/crux-and-rum-differences/) recently to understand why you may see differences in these datasets.
+ : - [Jeremy Wagner](https://twitter.com/malchata) wrote a guide on [Optimizing Long Tasks](https://web.dev/optimize-long-tasks/), which can help you improve your FID and INP metrics.
+
```

Comment:
I think these updates are a better fit for the newsletter-style CrUX Announce post, but since they're not directly relevant to any product updates we should omit them from the change log.

Reply:
```suggestion
```